models/ironbank/ethereum/ironbank_ethereum_itokens.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(
+    materialized = 'view',
```

Comment:
this property isn't necessary as it's already in the dbt_project yaml file you added. it's only necessary if you override that value.

Reply:
Got it. Fixed.